### ISSUE TITLE
Isolated post-capillary pulmonary hypertension

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6494]
**Preferred term label:**  
Isolated post-capillary pulmonary hypertension

**Synonyms**

**Definition (free text, please give PubMed ID)**
Post-capillary pulmonary hypertension is a haemodynamic condition characterised by elevated mean pulmonary artery pressure (mPAP > 20 mmHg)  and pulmonary artery wedge pressure (PAWP >15 mmHg) and pulmonary vascular resistance (PVR< 3 WU) PMID: 30545968

**Parent term (use hpo.jax.org/app)**
Elevated pulmonary artery pressure HP:0004890
(I would suggest moving pulmonary venous hypertension HP:0030950 as a child term of post-capillary pulmonary hypertension)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Pulmonary hypertension with unclear multifactorial mechanisms ORPHA:275844
Pulmonary hypertension associated with chronic hemolytic anaemia ORPHA:275828

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik Should "postcapillary" still be a child of Elevated pulmonary artery pressure HP:0004890?
The definition appears to be for "precapillary", please check this.

**Comment by eswietlik** [comment]
@pnrobinson Thanks, well spotted. The definition is ok, the name was wrong. Corrected now.

Pre-capillary pulmonary hypertension is a haemodynamic condition characterised by elevated mean pulmonary artery pressure (mPAP > 20 mmHg) and pulmonary vascular resistance (PVR ≥ 3 WU)  accompanied by normal pulmonary artery wedge pressure (PAWP ≤ 15 mmHg) PMID: 30545968

Post-capillary pulmonary hypertension is a haemodynamic condition characterised by elevated mean pulmonary artery pressure (mPAP > 20 mmHg) and pulmonary artery wedge pressure (PAWP >15 mmHg) and pulmonary vascular resistance (PVR< 3 WU) PMID: 30545968

The difference is

**Comment by pnrobinson** [comment]
@eswietlik  -- could we provide more context? I do not understand why elevated mean pulmonary artery pressure is POST-capillary.

**Comment by eswietlik** [comment]
@pnrobinson Post-capillary hypertension resulting from left heart disease causes backpressure and increases the hydrostatic pressure in the pulmonary veins which eventually results in increased pressure in pulmonary arteries.

**Comment by pnrobinson** [comment]
added
